browser/src/VariantPage/VariantTranscriptConsequences.tsx

```diff
@@ -13,7 +13,10 @@ type Props = {
 }
 
 const VariantTranscriptConsequences = ({ variant }: Props) => {
-  const { transcript_consequences: transcriptConsequences } = variant
+  let { transcript_consequences: transcriptConsequences } = variant
```

Comment:
Let's do `const transcriptConsequences = variant.transcript_consequences || []`. In my experience you really want to avoid using `let` if there's any reasonable alternative.

Reply:
Fixed